exercises/gigasecond/introduction.md

```diff
@@ -0,0 +1,20 @@
+# Introduction
+
+The way we measure time is kind of messy.
+We have 60 seconds in a minute, and 60 minutes in an hour.
+This comes from ancient Babylon, where they used 60 as the basis for their number system.
+We have 24 hours in a day, 7 days in a week, and months?
+Well, for months it depends not only on which month it is, but also what calendar is used in the country you live in.
+
+If we ever colonize Mars or some other planet, it's going to get even messier.
+If someone says "year" do they mean Earth-year or Mars-year?
+
+Instead, we could use the metric system, and count everything in seconds.
+
+- A food recipe might explain that you need to let the brownies cook in the oven for two kiloseconds (that's two thousand seconds).
+- Perhaps you and your family would travel to somewhere exotic for two megaseconds (that's two million seconds).
+- And if you and your spouse were married for _a thousand million_ seconds, you would celebrate your one gigasecond anniversary.
```

Comment:
I love all these examples.   😄   Given that each programming language handles these time "shenanigans" differently (see Python's [timedelta](https://docs.python.org/3.9/library/datetime.html#timedelta-objects) for some fun and games), I am thinking a comment/note encouraging students to look at how their programming language deals with time might be a good addition.  That said, perhaps its better to encourage each language track to do an instruction append for it. 🤔

Reply:
I think the instruction append is the way to go here. I've made a note in my pile of problem-specs notes, so that we can come back to it once we get a handle on this first pass of docs changes.